src/ui/component/Ecology/EcologyNavBar.tsx

```diff
@@ -25,9 +26,9 @@ export const EcologyNavBar = ({ className, style, chainId }: Props) => {
   return (
     <div
       className={clsx(
-        'flex items-center px-[20px] py-[10px] min-h-[48px]',
+        'flex items-center px-[20px] py-[10px] min-h-[48px] z-[100]',
         'border-b-[0.5px] border-rabby-neutral-line',
-        'bg-r-neutral-bg1',
+        'bg-rabby-neutral-bg1',
```

Comment:
why changing here?

Reply:
Was unsure if there's a specific reason you used -r- and not -rabby- since -r- is awkward for when you do things like border-r- as it's also used to signify "right"

Reply:
revert please, `-rabby-` is not a valid var name for our color
also prev line `z-[100]` if not useful, please revert